Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/BaseInteractiveElement.cs

```diff
@@ -90,6 +97,35 @@ private void Awake()
             SetStateOn(DefaultStateName);
         }
 
+        public virtual void Start()
+        {
+            // If the SelectFar state is in the States list at start and the Global property is true, then
```

Comment:
I would expect that the control would be registered as a global handler if the global property is on, even if SelectFar is not in the states list. Any reason we are requiring SelectFar to be in the states list? That seems like a hidden requirement.

Reply:
The reason it is specific to the SelectFar state is because that global property is associated with SelectFar state. 

![image](https://user-images.githubusercontent.com/53493796/101222132-fd916000-363d-11eb-9cf7-f4216e8860db.png)